MuchToDoWithoutAdo/app/src/main/java/com/scottlindley/muchtodowithoutado/RecyclerViewAdapters/HomeRecyclerViewAdapter.java

```diff
@@ -0,0 +1,66 @@
+package com.scottlindley.muchtodowithoutado.RecyclerViewAdapters;
+
+import android.content.Context;
+import android.content.Intent;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.scottlindley.muchtodowithoutado.Activities.ToDoListActivity;
+import com.scottlindley.muchtodowithoutado.JavaObjects.ToDoList;
+import com.scottlindley.muchtodowithoutado.R;
+import com.scottlindley.muchtodowithoutado.ViewHolders.HomeScreenViewHolder;
+
+import java.util.ArrayList;
+
+import static com.scottlindley.muchtodowithoutado.Activities.HomeActivity.mData;
```

Comment:
This works but it's not a good idea. This class should not depend on specific variables in another class. For example, if you ever decide to change the name of that variable in MainActivity, it would break this class. It's much better to "de-couple" the classes by having this class's constructor take in a list of `ToDoList`s as an argument, then save a reference to your `mHomeList` member variable.